client/state/geolocation/reducer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External Dependencies
+ */
+import { combineReducers } from 'redux';
+import camelCase from 'lodash/camelCase';
+import mapKeys from 'lodash/mapKeys';
```

Comment:
Optional, but we can now do the following without worrying about bundle size: see #6539
```
import { camelCase, mapKeys } from 'lodash';
```